src/config/queue.ts

```diff
@@ -55,6 +55,12 @@ async function initializePatientQueue(conn: Connection) {
     const exchangeName = "patient_list";
     let channel: Channel;
 
+    var oldZipMap = new Map<Number, Number>();
```

Comment:
In Javascript, the default variable declaration type is usually `const`. Even if the variable is a data structure and we are appending, removing, etc. to that structure, it can still be a constant because we are not changing the variable but the content of the variable.

Reply:
This can be applied to other variable declarations too.